src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,36 @@
 package core.basesyntax;
 
-public class Car {
+/**
+ * Don’t use super.clone().
+ * Use constructor to create new objects.
+ * Use fields itself instead of getters.
+ * Don’t use this. keyword. This is redundant.
+ * Don’t create redundant variables.
+ */
```

Comment:
it's  cool that you are using checklist, but let's not add it to your code.(Because it's not a part of java documentation)

Reply:
done!